openpype/pipeline/colorspace.py

```diff
@@ -0,0 +1,213 @@
+from copy import deepcopy
+import re
+import os
+import json
+import platform
+import contextlib
+import tempfile
+from openpype import PACKAGE_DIR
+from openpype.settings import get_project_settings
+from openpype.lib import (
+    StringTemplate,
+    run_openpype_process
+)
+from openpype.pipeline import Anatomy
+from openpype.lib.log import Logger
```

Comment:
There is no need to import explicitly from `openpype.lib.log`
```suggestion
from openpype.lib import (
    StringTemplate,
    run_openpype_process,
    Logger
)
from openpype.pipeline import Anatomy
```